Manifest.toml

```diff
@@ -34,16 +34,16 @@ deps = ["Random", "Serialization", "Sockets"]
 uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 
 [[GitHub]]
-deps = ["Base64", "Compat", "Dates", "HTTP", "JSON", "MbedTLS", "Test"]
-git-tree-sha1 = "b988ce46c9b682acfeed4801cbaafa1ef954ad31"
+deps = ["Base64", "Compat", "Dates", "HTTP", "JSON", "MbedTLS", "Sockets", "Test"]
+path = "/home/tan/.julia/dev/GitHub"
```

Comment:
Did you mean to put a local absolute path in here?

Reply:
No. I was waiting for GitHub.jl to be tagged. I should have marked this as WIP. My bad.

Reply:
No worries, just thought I'd check!